net.pdl2ork.PdL2Ork.yml

```diff
@@ -0,0 +1,199 @@
+id: net.pdl2ork.PdL2Ork
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+default-branch: stable
+command: pd-wrapper.sh
+finish-args:
+# Display
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
```

Comment:
does it support wayland? If not, this should be removed. Otherwise line 10 is `fallback-x11`

Reply:
It uses node webkit which I think is in part reliant on the wayland.

Reply:
I don't think so.

Reply:
Thanks. Would this make the UI (e.g. mouse cursor, etc.) look different on Wayland desktops (e.g. Ubuntu) if we fell back to X11? For reference, other similar forks have this kind of setup, although they also did not have a release in a while.

Reply:
Should I also remove line 11?

Reply:
just change `x11` to `fallback-x11`.